app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -249,7 +249,8 @@ $top-border: solid 2px $grey-3;
   }
 
   &:hover {
-    .gem-c-step-nav__button {
+    .gem-c-step-nav__button,
+    .gem-c-step-nav__circle {
```

Comment:
Although this is a bit clearer and more consistent with how other links look and behave, it is hardly noticeable as the text that is changed is usually just a single tiny character/number. I wonder if it should be changed (to e.g. changing the background colour or the colour of the circle border). Although that means you'd have to decide what's more important: noticeability and consistency? Or is there a way both could be achieved?

Reply:
Yeah, good question. I struggled with this one for a bit - we had no good design ideas. Eventually I decided that while only a small change, it was at least a small improvement. We may have to revisit this in future.